### ISSUE TITLE
Implement internet connectivity check

### ISSUE THREAD

**Issue by esQmo** [michaldrabik/showly#101]
Actually the app only checks if the device is connected to either WiFi or data connection. Would be great to check for connectivity instead: if the actual network has connectivity, has access to internet. Do you think you can add this too?

**Comment by michaldrabik** [comment]
The internet connectivity should be checked using:

https://github.com/michaldrabik/showly-2.0/blob/58ef9630a2d0ca3ab3470667795d0cacffff4466/app/src/main/java/com/michaldrabik/showly2/utilities/NetworkMonitorCallbacks.kt#L33

Did you actually stumbled upon different case?

**Comment by esQmo** [comment]
Sorry, didn't get the notification. Well the callbacks work when the device isn't connected to a network (wifi or data connection). But what I'm pointing is the fact that the app doens't check if the actual connection has internet connectitiy. Tried on a Wifi hotspot and data connection (4G) with no internet and the callbacks is not triggered, no message is being shown

**Comment by michaldrabik** [comment]
Thanks. I will give it a second look then. I think it's not that fatal issue anyway as this case is quite rare I suppose. And in that case you will still get a standard error If you try to do an action that requires internet.

Thanks!

**Comment by esQmo** [comment]
Indeed, this is not a big issue anyway. I just thought it would be nice if that feature was implemented, like in Netflix (however I only checked for low speed connection).
